hackathon_site/hardware/serializers.py

```diff
@@ -207,6 +208,16 @@ def create(self, validated_data):
         )
         new_order = None
         response_data = {"order_id": None, "hardware": [], "errors": []}
+
+        # The reason why doing this is because the id field stores the hardware object, django cannot translate hardware object into JSON. Therefore, loop has been used to call Profile seriliazer and assign the JSON file to the validated_data filed.
```

Comment:
I'm not exactly sure where the error you're getting is coming from. Delete this block of code, and everything should work - it does on my machine

Reply:
Seen above

Reply:
Okay, this makes sense, could you change "Profile serializer" to "Hardware serializer" in the comment